wallet-server/src/main/java/com/android/identity/wallet/server/FlowServlet.kt

```diff
@@ -0,0 +1,91 @@
+package com.android.identity.wallet.server
+
+import com.android.identity.flow.handler.FlowHandlerLocal
+import com.android.identity.issuance.hardcoded.IssuingAuthorityState
+import com.android.identity.issuance.hardcoded.ProofingState
+import com.android.identity.issuance.hardcoded.RegistrationState
+import com.android.identity.issuance.hardcoded.RequestCredentialsState
+import com.android.identity.issuance.hardcoded.register
+import kotlinx.coroutines.runBlocking
+import jakarta.servlet.http.HttpServlet
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import kotlinx.io.bytestring.ByteString
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import java.security.Security
+
+// To run this servlet for development, use this command:
+//
+// ./gradlew wallet-server:tomcatRun
+//
+// This should start the server on http://localhost:8080/wallet-server. To get the Android
+// app to connect it, use this command to tunnel 8080 port local connections on the Android
+// device to your development machine:
+//
+// adb reverse tcp:8080 tcp:8080
+//
+class FlowServlet : HttpServlet() {
```

Comment:
This is fine for now, down the road I think we want something like a top-level `dev-server` module which provides wallet-server, cloud-secure-area, reader-server end-points to be used only for development... this should use a single configuration so end-points can share configuration.